scripts/cli/main.py

```
@@ -0,0 +1,804 @@
+                new_lines.append(line)
+
+            if changed:
+                files_changed += 1
+                rel_path = yml_file.relative_to(PROJECT_ROOT)
+                print(f"{'🔍' if not args.apply else '✅'} {rel_path}")
+
+                if args.apply:
+                    with open(yml_file, 'w', encoding='utf-8') as f:
+                        f.writelines(new_lines)
+
+        except Exception as e:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)